ember/package.json

```diff
@@ -62,6 +62,8 @@
     "ember-promise-helpers": "^1.0.9",
     "ember-qunit": "^4.6.0",
     "ember-resolver": "^8.0.0",
+    "ember-simple-auth": "^3.0.1",
+    "ember-simple-auth-oidc": "^3.0.1",
```

Comment:
What is the reason here for specifying `ember-simple-auth`? Isnt this a child dependency from `ember-simple-auth-oidc`?

Reply:
Right, I removed the `ember-simple-auth` from the `package.json`